dashboard/static/js/file_access_within_week.js

```diff
@@ -0,0 +1,301 @@
+var highlightData = function(file_name) {
```

Comment:
We really should namespace these javascript functions perhaps with revealing module. If you'd like create another ticket and I can clean this up. ;)

 https://addyosmani.com/resources/essentialjsdesignpatterns/book/#revealingmodulepatternjavascript

Reply:
Another ticket would be good once JS files are all done.